src/libp2p/nuget.config

```diff
@@ -3,14 +3,14 @@
   <packageSources>
```

Comment:
The entire file can be dropped safely

Reply:
(if we update libp2p.address in the libp2p.core to production version)

Reply:
For central package management, it is recommended to have.